### ISSUE TITLE
st.labs

### ISSUE THREAD

**Issue by subasish** [ehrlinger/ggRandomForests#25]
The `package` and the `vignette` codes are working perfectly except `st.labs` involved codes.

> plot(gg_md, lbls=st.labs)
Error in plot.gg_minimal_depth(gg_md, lbls = st.labs) : 
  object 'st.labs' not found

**Comment by 22csnyder** [comment]
I noticed this as well

**Comment by 22csnyder** [comment]
Oh I just found this line in one of the vignettes. It's in the code RandomForestSRC-Survival.Rnw  but not in the pdf

dta.labs <- data.frame(cbind(names = colnames(pbc), label = labels, type = cls))
# Put the "years" variable on top.
dta.labs <- rbind(dta.labs[nrow(dta.labs),], dta.labs[-nrow(dta.labs),])
st.labs <- as.character(dta.labs$label)

Honestly I haven't gotten it to work, but I'm a complete beginner. 
2 days new at R

**Comment by ehrlinger** [comment]
Right. I define st.labs in the code in the vignette, but not out where everyone can see it. 

The new rmarkdown release has a "hide code" option that looks nice for the html docs, but not pdflatex... so that's not the way to go about it. I may just remove the "nice" labels that st.labs provides from the document all together, because they are confusing.

Still thinking about the correct approach... in the mean time, here is the st.labs definition you'll need to reproduce the plot:

```
labels <- c("Event (F = censor, T = death)", 
            "Treament (DPCA, Placebo)", 
            "Age (years)", 
            "Female = T", 
            "Presence of Asictes", 
            "Presence of Hepatomegaly", 
            "Presence of Spiders", 
            "Edema (0, 0.5, 1)", 
            "Serum Bilirubin (mg/dl)", 
            "Serum Cholesterol (mg/dl)", 
            "Albumin (gm/dl)", 
            "Urine Copper (ug/day)", 
            "Alkaline Phosphatase (U/liter)", 
            "SGOT (U/ml)", 
            "Triglicerides (mg/dl)", 
            "Platelets per cubic ml/1000", 
            "Prothrombin time (sec)", 
            "Histologic Stage", 
            "Time (years)")

dta.labs <- data.frame(cbind(names = colnames(pbc), label = labels, type = cls))
# Put the "years" variable on top.
dta.labs <- rbind(dta.labs[nrow(dta.labs),], dta.labs[-nrow(dta.labs),])

st.labs <- as.character(dta.labs$label)
names(st.labs) <- rownames(dta.labs)
```

This section in the vignette dos a lot of data prep work. I might need to do this outside of the vignette, and store the updated data set for distribution with the `ggRandomForests` package.

**Comment by ehrlinger** [comment]
The vignette has been rewritten in v2.7.x with all setup code inline. `st_labs` is now defined within the survival vignette before it is used. Closing; please reopen against v2.7.2 if the issue still reproduces.
